scripts/migration/migrate_retraction_dates.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Script to migrate retracted registrations so that their date modified is date of retraction."""
+
+import sys
+import logging
+
+from modularodm import Q
+from website.models import Node, NodeLog
+from website.app import init_app
+from scripts import utils as script_utils
+from framework.transactions.context import TokuTransaction
+
+logger = logging.getLogger(__name__)
+
+
+def do_migration(logs):
+    # ... perform the migration using a list of logs ...
+    for log in logs:
+        registration_id = log.params.get('registration')
+        if registration_id:
+            registration = Node.load(registration_id)
+            registration.date_modified = log.date
```

Comment:
The most recent change on the offending node is from a `made_public` `NodeLog`, presumably because it was embargoed. Please handle this case too.